config.ini

```diff
@@ -66,7 +66,7 @@ lhs_params=[[[-3, 3], [-3, 3], [-3, 3]], [[-3, 3], [-3, 3], [-3, 3]], true]
 ; the subgroup of postprocessing functions
 postproc_funcs_filter=["safe_inverse", "lambda x: x", "lambda x: x**2", "lambda x: safe_inverse(x**2)", "safe_sqrt", "lambda x: safe_inverse(safe_sqrt(x))"]
 
-; options for hastable_file_opeartion: use, generate, expand
+; options for hastable_file_operation: use, generate, expand

```

Comment:
My fix was to the misspelling of "operation" (it was originally "opeartion"). I now notice that "hastable" should probably be "hashtable," too.

Reply:
Oh, missed the ar<->ra. Will fix the hastable too. Thanks!